esp-link/main.c

```diff
@@ -96,6 +97,8 @@ HttpdBuiltInUrl builtInUrls[] = {
   { "/services/info", cgiServicesInfo, NULL },
   { "/services/update", cgiServicesSet, NULL },
   { "/pins", cgiPins, NULL },
+  { "/telnet/info", cgiTelnet, NULL},
+  { "/telnet/update", cgiTelnet, NULL},
```

Comment:
You only need one since you distinguish using GET and POST. Not sure why there are two for services, something for me to check.